manifests/addons/gen.sh

```diff
@@ -81,3 +83,12 @@ function compressDashboard() {
     --from-file=istio-mesh-dashboard.json="${TMP}/istio-mesh-dashboard.json" \
     --from-file=istio-extension-dashboard.json="${TMP}/istio-extension-dashboard.json"
 } > "${ADDONS}/grafana.yaml"
+
+# Set up loki
+{
+  helm3 template loki loki \
+    --namespace istio-system \
```

Comment:
this line does not work because of the implement of [loki chart](https://github.com/grafana/loki/tree/main/production/helm/loki)

users should apply with following command:
```console
kubectl -f samples/addons/loki.yaml -nistio-system
```